src/cryptonote_core/cryptonote_core.h

```diff
@@ -117,7 +117,7 @@ namespace cryptonote
       *
       * @return true if we haven't seen it before and thus need to relay.
       */
-     bool handle_uptime_proof(const NOTIFY_UPTIME_PROOF::request &proof);
+     bool handle_uptime_proof(const NOTIFY_UPTIME_PROOF::request &proof, bool *my_uptime_proof_confirmation);
```

Comment:
It doesn't look like this argument is optional (i.e. can't pass in nullptr), so perhaps would be cleaner as a (non-const) `bool &` instead of a pointer?

Reply:
Changed